src/admin-events/admin-events.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Voting" : {
+    "OwnershipTransferred" : {"type" : "Suspicious", "severity" : "High"},
+    "PriceRequestAdded" : {"type" : "Unknown", "severity" : "Low"},
+    "PriceResolved" : {"type" : "Unknown", "severity" : "Low"},
+    "RewardsRetrieved" : {"type" : "Unknown", "severity" : "Low"},
+    "VoteCommitted" : {"type" : "Unknown", "severity" : "Low"},
+    "VoteRevealed" : {"type" : "Unknown", "severity" : "Low"}
+  },
+  "Governor" : {
+    "NewProposal" : {"type" : "Unknown", "severity" : "Low"},
+    "ProposalExecuted" : {"type" : "Unknown", "severity" : "Low"},
+    "AddedSharedMember" : {"type" : "Unknown", "severity" : "Low"},
```

Comment:
Maybe AddedSharedMember and RemovedSharedMember should be promoted to type "suspicious"?

Reply:
Done